assets/js/components/DeregistrationModal/DeregistrationModal.jsx

```diff
@@ -22,6 +23,14 @@ function DeregistrationModal({
         discovered by the agent in this host, including the host itself and any
         other component depending on it.
       </div>
+      {isError && (
+        <div
+          className="bg-red-100 border border-red-400 text-red-700 px-4 py-3 mt-3 rounded relative"
+          role="alert"
+        >
+          Error occurred when requesting deregistration of host
```

Comment:
Please extract this message to a constant ad use as default message for another prop called errorMessage, so we could have this granularity, and if we have to provide a more appropriate message for a specific case we could do it.